etc/databases/world/updates/updates.sql

```diff
@@ -30682,5 +30682,27 @@ begin not atomic
 
         insert into applied_updates values ('110320231');
     end if;
+
+    -- 12/03/2023 1
+    if(select count(*) from applied_updates where id = '120320231') = 0 then
+
+        -- Set Coldridge Mountaineer's movement type to idle instead of wandering.
+        update spawns_creatures set movement_type = 0 where spawn_id = 398;
```

Comment:
Please provide proof for this so we can verify it. :pray:

Reply:
Is him constantly running through a tree proof enough that he's not supposed to wander? He's a guard, they usually don't do that.

Reply:
According to the sniffs, he wanders:
```
+-----+---------+------------+------------+------------+-----------------+
| id  | guid    | position_x | position_y | position_z | wander_distance |
+-----+---------+------------+------------+------------+-----------------+
| 853 | 1896261 |   -6239.11 |    338.359 |    382.997 |         4.81431 |
+-----+---------+------------+------------+------------+-----------------+
```
In fact he should wander for 5 yards instead of 3 (current value). With namigator, it shouldn't run through trees.

Reply:
Okay, let him wander but for 5 yards then.